src/terra-dev-site/guide/DockerDevelopmentEnvironment.i.guide.md

```diff
@@ -0,0 +1,91 @@
+## Docker Development Environment
+
+Terra provides a docker development environment that allows developers access to a common environment to develop Terra components regardless of the OS they are developing on.
+
+In each Terra repository there is a docker-compose.yml file that defines several services, most importantly ```dev``` and ```test```.
+
+To build all the services in the docker-compose file, run:
+```
+docker-compose build
+```
+
+### Dev
+The dev service provides a linux environment with node and npm pre-installed. By default the dev service volume mounts the entire code repo into the docker container to enable hot reloading during development.
```

Comment:
mounts the entire ~code~ **local repo directory** repo into the docker container. 

To enable hot reloading allow continuous development without rebuild the docker image

Reply:
Hot reloading implies to me that you don't have to rebuild (or recompile)